example/README.MD

```diff
@@ -0,0 +1,17 @@
+# Getting started
+
+# Android
+
+To get this project running on Android just follow these steps
```

Comment:
Suggest: To get this project *start your Android emulator, make sure not other metro packagers are running* and then just follow these steps

(esp the emulator start, I wish android just started)

and as for the packager mention, that still trips me up about once a day ;-)